gear-programs/vara-tokenizer/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "vara-tokenizer"
+version.workspace = true
+edition.workspace = true
+
+[dependencies]
+vara-tokenizer-app = { path = "app" }
```

Comment:
I would place that and all its deps (client, etc) in the workspace's Cargo.toml and use like `.workspace = true`

Reply:
It is from `sails` template. Main crate referenced in the workspace's Cargo.toml. The `-app' crate is intended for internal use only.

Reply:
But it makes sense if you're referring to a service from another.

Reply:
@gshep @vobradovich What's with this review?

Reply:
all crates added to workspace's Cargo.toml